### PR TITLE
[TEVA-3541] Update disaster recovery documentation

### PR DESCRIPTION
**Pull request by felixtheflex**
## Jira ticket URL

- Just add the ticket number to the end:

https://dfedigital.atlassian.net/browse/TEVA-3541

Update disaster recovery documentation to include:-

    - Documentation on how to start disaster recovery simulatation

    - Add instructions on how to delete old database instance

    - Improve documentation on how to switch web app to `MAINTENANCE_MODE`

    - Add instructions about keeping non-technical stakeholders abreast via #tv_product channel

### CODE REVIEW

File: documentation/maintenance-mode.md [hunk 1]
```diff
@@ -4,7 +4,15 @@ The application has a simple routing-level maintenance mode triggered by the `MA
 environment variable. This needs the app to be restarted to enable maintenance mode, e.g. via
 a deploy or a `cf restage`. This is handy in case of a critical bug being discovered where we need
 to take the service offline, or in case of maintenance where we want to avoid users interacting
-with the service while
+with the service.
 
 When enabled, all requests of all types will be routed to the maintenance page (found under
 `app/views/errors/maintenance.html.erb`).
+
+Do the following enable maintenance mode:
```
Comment (josephhull676) [inline documentation/maintenance-mode.md:12]: to enable!

File: documentation/maintenance-mode.md [hunk 2]
```diff
@@ -4,7 +4,15 @@ The application has a simple routing-level maintenance mode triggered by the `MA
...
+
+Login to PaaS: `cf login --sso`
+
+set maintenance mode environment variable: `cf set-env teaching-vacancies-dev MAINTENANCE_MODE 1`
+
+restage app: cf restage: `cf restage teaching-vacancies-dev`
```
Comment (josephhull676) [inline documentation/maintenance-mode.md:18]: I'd assume we'd want to switch off maintenance mode at some point. How do we do that? Same command with just a different value for MAINTENANCE_MODE? I assume the value you're giving above is '1'?

**Review by josephhull676 — COMMENTED** [review]
I seem to remember that a few things weren't documented yesterday when we did the simulation: 

- That we need the 'deployments' role?
- I remember this: `aws-vault exec Deployments -- make dev remove-postgres-tf-state` we had to prefix `make dev remove-postgres-tf-state` with `aws-vault exec Deployments --`. I don't know why but perhaps worth mentioning?
- How to delete the corrupted database instance.

File: documentation/maintenance-mode.md (hunk 2 above)
Comment (felixtheflex) [reply documentation/maintenance-mode.md:18]: not quite, it will be taken off automatically in this scenario ...

**Review by josephhull676 — APPROVED** [review]

**Pull request merged**